mlrun/datastore/targets.py

```diff
@@ -1190,6 +1193,26 @@ def get_table_object(self):
             flush_interval_secs=mlrun.mlconf.feature_store.flush_interval,
         )
 
+    def get_spark_options(self, key_column=None, timestamp_key=None, overwrite=True):
+        parsed_url = urlparse(self.get_target_path())
+        if parsed_url.hostname is None:
+            self.path = mlrun.mlconf.redis.url
+            parsed_url = urlparse(self.get_target_path())
```

Comment:
not  parsed_url = urlparse(self.path)?

Reply:
In all the places we use ' urlparse(self.get_target_path())'. In this case it probably returns the self.path. But I wanted to keep it generic